lib/Parser/rterrors.h

```diff
@@ -260,7 +260,7 @@ RT_ERROR_MSG(JSERR_IntlNotImplemented, 5126, "", "Intl operation '%s' is not imp
 
 RT_ERROR_MSG(JSERR_ArgumentOutOfRange, 5130, "%s: argument out of range", "argument out of range", kjstRangeError, 0)
 RT_ERROR_MSG(JSERR_ErrorOnNew, 5131, "", "Function is not a constructor", kjstTypeError, 0)
-RT_ERROR_MSG(JSERR_This_NeedArrayIterator, 5132, "%s: 'this' is not an Array Iterator object", "Array Iterator expected", kjstTypeError, 0) // {Locked="\'this\'"}
+RT_ERROR_MSG(JSERR_This_NeedArrayIterator, 5132, "%s: 'this' is not a Array Iterator object", "Array Iterator expected", kjstTypeError, 0) // {Locked="\'this\'"}
```

Comment:
an Array instead of a Array?

Reply:
yeah, I know :(  but the default platform.raiseNeedObjectOfType construcs an error message with 'a' instead of 'an', and I need this change to keep it uniform when jsBuiltins are enabled/disabled.

Reply:
It should be pretty easy to just add another built in to raise this error directly (raiseThis_NeedArrayIterator) instead of raiseNeedObjectOfType?

Reply:
Then the error number that is set on the Error object will be the same